lib/addons/karpenter/index.ts

```diff
@@ -5,9 +5,11 @@ import { ClusterInfo } from '../../spi';
 import { conflictsWith, createNamespace, createServiceAccount, dependable, setPath, } from '../../utils';
 import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from '../helm-addon';
 import { KarpenterControllerPolicy } from './iam';
+import { CfnOutput } from 'aws-cdk-lib';
 import * as md5 from 'ts-md5';
 import * as semver from 'semver';
 import * as assert from "assert";
+import { Instance } from 'aws-cdk-lib/aws-ec2';
```

Comment:
(minor): please fix this unused import warning.

Reply:
addressed